### PR TITLE
gatsby + Prismic link/LinkResolver configuration

### PR DESCRIPTION
**Pull request by PaulinaVPG**

### CODE REVIEW

File: src/components/TopMenu.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ const TopMenu = ({ topMenu, activeDocMeta }) => {
   return (
     <header>
       <div className="menu">
-        <Link to="/">
+        <a href="/">
```
Comment (levimykel) [inline src/components/TopMenu.js:22]: Why did you change this to an anchor link?

File: src/utils/linkResolver.js [hunk 2]
```diff
@@ -1,21 +1,23 @@
-import { defaultLanguage } from '../../prismic-config'
+import { defaultLanguage } from "../../prismic-config";
```
Comment (levimykel) [inline src/utils/linkResolver.js:1]: Is there a reason you changed these to double quotes?

File: src/utils/linkResolver.js [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
-      : `/page/${doc.lang}/${doc.uid}`
+  if (properties.type === "page") {
+    return properties.lang === defaultLanguage
+      ? `/page/${properties.uid}`
+      : `/page/${properties.lang}/${properties.uid}`;
   }
 
-  return '/'
-}
+  return "/";
```
Comment (levimykel) [inline src/utils/linkResolver.js:20]: Change all these back to single quotes, and this page should be good.

File: src/components/TopMenu.js (hunk 1 above)
Comment (PaulinaVPG) [reply src/components/TopMenu.js:22]: I changed it because it was just redirecting to the homepage with a hardcoded "/", but since it is still a link between pages, i'm going to change it back to be a gatsby Link.

File: src/utils/linkResolver.js (hunk 2 above)
Comment (PaulinaVPG) [reply src/utils/linkResolver.js:1]: No particular reason.
